src/api/app/controllers/webui/interconnects_controller.rb

```diff
@@ -2,20 +2,40 @@ class Webui::InterconnectsController < Webui::WebuiController
   before_action :require_admin
 
   def new
-    @interconnect = RemoteProject.new(default_values)
     # TODO: Remove the statement after migration is finished
-    switch_to_webui2 if Rails.env.development? || Rails.env.test?
+    return if switch_to_webui2 && (Rails.env.development? || Rails.env.test?)
```

Comment:
The other configuration pages are already active since we merged #7097, so we should also remove ` && (Rails.env.development? || Rails.env.test?)` on this line and all other lines.